scripts/start-small

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
```

Comment:
There is a line in the script's README for this, it could do with removing please. Lines 19-21 https://github.com/nhsuk/pharmacy-data-etl/blob/master/scripts/README.md